src/shared/pretty-print.c

```diff
@@ -222,11 +222,11 @@ static int cat_file(const char *filename, bool newline, CatFlags flags) {
                 }
 
                 /* empty line */
-                if (isempty(l) && FLAGS_SET(flags, CAT_TLDR))
+                if (isempty(l) && FLAGS_SET(flags, CAT_TLDR) && !continued)
```

Comment:
what about comments, shouldn't they also be skipped after continuation lines? (just wondering, no idea what our conf file parser actually does there)